doc/guidelines/frontend-ui-usability-guide.md

```diff
@@ -0,0 +1,166 @@
+
+### User Interface (UI)
+
+> Set of all the components of an interactive system that provide information and controls for the user to accomplish specific tasks with the interactive system.
+
+### User Experience (UX)
+
+> Combination of user's perceptions and responses that result from the use and/or anticipated use of a system, product or service.
+
+## Guidelines
+
+### Text
+
+- **Alignment**
+
+  - Left align when text spans over more than three lines because centered text becomes hard to follow when having to jump to the next line multiple times.
+  - Avoid mixing two different alignments in a text section, e.g. a centered headline should be followed by centered text.
+
+- **Hierarchies / lists**
+
+  - Avoid using more than two different font sizes.
+  - Use bold text and colours for deeper nested hierarchies.
+
+- **Spacing / margin**  
+  Multiplier strategy to calculate the margins - if in doubt, use a multiplier of 2, e.g.:
+
+  - Text sections:  
+    headline-1 <- 16px -> text <- 32px -> headline-2
+  - Navigation bar:  
+    logo <- 48px -> navigation-link-1 <- 24px -> navigation-link-2 <- 24px -> navigation-link-3
+
+- **Line height**  
+  The bigger the text the smaller the line height's value should be.
+
+  - Headings: 1.1 - 1.3
+  - Running text: 1.3 - 1.5
+
+- **Letter spacing**  
+  Often times, like in PermaplanT's current case, the default value (in Tailwind: _tracking-normal_) is good enough.
+  - Headings: negative values can be used for the bigger headings h1-h3, i.e. _tracking-tighter_ or _tracking-tight_
+  - Running text: usually can be left as is but, if needed, can be set to _tracking-wide_ or _tracking-wider_
+
+### Colors
+
+PermaplanT uses following **three colour palettes**:
+
+- **Neutral** ~60% (PermaplanT: gray): **dominant** colour of the design, e.g. used for background, text and labels
+- **Primary** ~30% (PermaplanT: asparagus green): PermaplanT's **main brand** colour, e.g. used for buttons, hovering, highlighting focused input fields
+- **Secondary** ~10% (PermaplanT: sea blue): PermaplanT's **second brand** colour, e.g. used for highlighting a few UI elements (e.g. in the layers section) and action-texts
+
+**Only those** three colour palettes **should be used** throughout the whole application. The ratio of their current usage in PermaplanT is conforming nicely to the popular **60-30-10** design rule.
+
+### Forms
+
+#### Design
+
+- Display a **centered heading** if the form is representing a whole page, e.g. PermaplanT's forms to create maps and seeds.
+- Add a **capitalized label** above each field.
+- **Align** labels to the **left** of their corresponding fields.
+- Always explicitly **associate** the **label with** the belonging **element** by using the label's _for_ attribute (in React: _htmlFor_).
+  This enables focusing the input field by clicking/tapping on the label.
```

Comment:
Maybe add something like this (everywhere where rationale is given) to make the guideline easier to read:

```suggestion
  _Rationale:_ This enables focusing the input field by clicking/tapping on the label.
```